### ISSUE TITLE
how does the black/whitelist work?

### ISSUE THREAD

**Issue by itzMiney** [iChun/Morph#1159]
I don't get why it says both black and white in "blackwhiteListedMobs"
Should I remove one so it's only "blackListedMobs" or should I do something different?
Also I'm generally not sure if the format I use is right
Right now my config is like this:
```
S:blackListedMobs <
     ancientwarfarenpc:faction.archer
     ancientwarfarenpc:faction.archer.elite
     ancientwarfarenpc:faction.cavalry
     ancientwarfarenpc:faction.civilian
     ancientwarfarenpc:faction.civilian.female
     ancientwarfarenpc:faction.leader
     ancientwarfarenpc:faction.priest
     ancientwarfarenpc:faction.siege_engineer
     ancientwarfarenpc:faction.soldier
     ancientwarfarenpc:faction.soldier.elite
     >
```
Because those Ancient Warfare NPCs for some reason crash the game for everyone on my server that uses them and crashes the game for players nearby as well, repeatetly. As soon as they join they crash almost instantly when morphed into one of those mobs. The only fix I found till now was demorphing them in console as soon as they joined but I think just blacklisting them in general and clearing them from anyone who already has them works better

I use Morph Mod for 1.12.2

Also comments on what to do here / examples of formatting would have been nice .-.

**Comment by Pgeorgaklis21** [comment]
Have you found anything about it at all? I'm having the same problem except its for the tf2 mod and i keep getting errored config files when I start the server again

**Comment by iChun** [comment]
As Morph has just been rewritten and released for MC 1.16.5, I am clearing the issue tracker as I have not provided any support Morph for 1.12.2 and earlier. My apologies for not addressing your issue.

Also: from the other comments I had just closed, in case you were curious, apparently the config was broken the entire time, whoops! It should be working fine in the 1.16.5 version (and also use the resource location of the entity)
